telemetry/definitions/commonDefinitions.json

```diff
@@ -550,6 +550,36 @@
             "description": "The type of the user trigger to send request to CodeWhisperer service",
             "allowedValues": ["OnDemand", "AutoTrigger"]
         },
+        {
+            "name": "codewhispererFirstCompletionLatency",
+            "type": "double",
+            "description": "The latency from user performs the triggering CW action to plugin call GenerateCompletions API for the first recommendation"
```

Comment:
The time it takes for the response to be received after the plugin makes a first GenerateCompletions API call.